sdk/eventhub/Azure.Messaging.EventHubs/samples/Program.cs

```diff
@@ -15,6 +15,16 @@ namespace Azure.Messaging.EventHubs.Samples
     ///
     public static class Program
```

Comment:
It may be helpful to add an example to the help text with the identity sample parameters set.